tests/run_bencher_aggregated.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import sys
+import os
+import json
+import subprocess
+import statistics
+from loguru import logger as LOG
+import infra.bencher
+
+
+def run_performance_tests(iterations: int):
+    """Run performance tests N times and save individual results"""
+    LOG.info(f"Running performance tests {iterations} times...")
+
+    for i in range(1, iterations + 1):
+        LOG.debug(f"Running iteration {i} of {iterations}")
+        try:
+            subprocess.run(["./tests.sh", "-C", "perf", "-L", "perf"], check=True)
```

Comment:
`./tests.sh` adds some unnecessary overhead to the start of every run (several `pip install` calls). I think we should assume-or-force that we're in the right venv, and call `ctest` directly.